src/anurag/Mall.java

```diff
@@ -0,0 +1,55 @@
+package anurag;
+
+public class Mall {
+	int item;
+	String customName;
+	String visaCard ;
+	String cash;
+    String section;
+    String paymentMethod;
+    
+    void itemAddedToCart(int item) {
+		this.item = item;
+		}
+	void customerName(String customName ) {
+		this.customName=customName;
+		}
+	void paymentDoneUsing(String paymentMethod) {
+		this.paymentMethod = paymentMethod;
+		}
+	void needBill(boolean bill){ 
+		 if(bill==true) 
```

Comment:
here you can write only bill only no need to compare with true, if **bill** is true then it will comes under if condition. same for haveCarryBag()